core/src/apps/management/upload_res.py

```
@@ -79,6 +156,7 @@ async def upload_res(ctx: wire.Context, msg: ResourceUpload) -> Success:
             metadata = json.loads(msg.nft_meta_data.decode("utf-8"))
         except BaseException as e:
             raise wire.DataError(f"Invalid metadata {e}")
+        metadata_len = len(msg.nft_meta_data)
         if any(key not in metadata.keys() for key in NFT_METADATA_ALLOWED_KEYS):
             raise wire.DataError("Invalid metadata")
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)